src/nvim/api/vim.c

```diff
@@ -634,6 +634,62 @@ Object nvim_call_function(String fn, Array args, Error *err)
   return _call_function(fn, args, NULL, err);
 }
 
+/// nvim_ex_call_function({noautocmd = true}, my_lua_cb, x, y, z)
+///
+/// local callable = vim.ex_wrap({autocmd = false}, my_lua_cb)
+/// callable(x, y, z)
+/// local wrapped = vim.
+//
+// nvim_ex_call_function({bufdo = {1, 3, 7, 8}}, my_lua_cb, ...)
+Object nvim_ex_call_luaref(Dictionary opts, LuaRef cb, Array args, Error *err)
```

Comment:
this can just be called `nvim_ex_call` or something without "lua" in the name.. The documentation will automatically indicate that it is "lua only" for now, and we can extend it later to rpc and/or vimscript.